### PR TITLE
Reveal links to user stats + groups blog post

### PR DESCRIPTION
**Pull request by goplayoutside3**
(This will be deployed on Tues Sept 17)

## Package
lib-content
lib-user

## Linked Issue and/or Talk Post
MyGroups related Slack convo: https://zooniverse.slack.com/archives/C010QAPB67J/p1726253388546699

## Describe your changes

- Reveal the FAQ question about how to get a volunteer certificate
- Add blog post link to Get Involved > Educate
- Add 'Learn More About Groups' to MyGroups on the signed-in homepage and `/groups` page.

## How to Review

- Note that you won't actually be able to see the wordpress blog until it's published after launch Tuesday, but hovering on the link should correctly preview the url for you.
- lib-content shows the FAQ and Educate page
- You have to run app-root locally and sign in to see MyGroups

# Checklist
_PR Creator - Please cater the checklist to fit the review needed for your code changes._
_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._

## General
- [ ] Tests are passing locally and on Github
- [ ] Documentation is up to date and changelog has been updated if appropriate
- [ ] You can `yarn panic && yarn bootstrap` or `docker-compose up --build` and FEM works as expected
- [ ] FEM works in all major desktop browsers: Firefox, Chrome, Edge, Safari (Use Browserstack account as needed)
- [ ] FEM works in a mobile browser

### CODE REVIEW

File: packages/lib-content/src/screens/Educate/Educate.js [hunk 1]
```diff
@@ -202,7 +201,7 @@ function Educate() {
                     // "See this <0>blog post</0> for details."
                     components={[
                       <Anchor
-                        key='group-stats-announcement'
+                        key='stats-in-classrooms-blog'
                         href='' // Need url after blog post
```
Comment (goplayoutside3) [inline packages/lib-content/src/screens/Educate/Educate.js:205]: This is an October update. See https://docs.google.com/spreadsheets/d/10k8yyZsRjaJens7BdffD6VUpoInZNLEPNQNgKSSTgpM/edit?disco=AAABVyWg6nA

**Review by kieftrav — APPROVED** [review]
LGTM - everything showed up where it should and all links match

**Comment by kieftrav** [comment]
See the below screenshots demonstrating behavior from running `app-root` locally & what I tested / looked for:

- Educators page shows link
- Homepage Groups section shows link to blog post
- Groups page shows link to blog post
- FAQ page explains link to volunteer certificate

educators
groups-home
groups-groups
volunteer-certificate

**Comment by seanmiller26** [comment]
Just noting the discrepancy in positioning of the 'Learn more about Groups'. Ok for launch, but we should consider a revisit at another time for consistency and to match the empty state wording.

**Comment by goplayoutside3** [comment]
@seanmiller26 the 'Learn more about groups' and 'Create a new group' position on the homepage correctly matches the Figma design. However, the 'Create a new group' button on a user's MyGroups _page_ has always existed in the lower left corner. Best to ask @mcbouslog if that was intentional or if maybe it just wasn't caught as incorrect placement during internal review rounds.

I'll note that I'm of the strong opinion that these pages are launching tomorrow, and I don't want to change the positioning of buttons post-launch to avoid user confusion.

**Comment by mcbouslog** [comment]
I wasn't aware the 'Create a new group' button on a user's MyGroups page has moved, when did that happen? It was in the bottom left in initial designs. Is that a change that needs to be made today? That's fine if so, should be a quick change.

**Comment by seanmiller26** [comment]
It probably was just missed around the time of internal reviews. No worries, @mcbouslog, but if its easy enough to match the placement of both links before launch, that would be great!

**Comment by mcbouslog** [comment]
Sounds good; I'll work on a related PR to move the group create button and learn more link on a user's MyGroups now.

**Comment by goplayoutside3** [comment]
Thanks all!

**Pull request merged**